src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
```

Comment:
Each time this method is called, new ColorSupplier and Random instanced are created. Let's think about how we can optimize this logic and create each instance only once.